scripts/setup-remote-cluster.sh

```
@@ -167,6 +167,9 @@ EOF
 oc ${hub} create secret generic osac-remote-kubeconfig \
     --from-file=kubeconfig="${REMOTE_KUBECONFIG_FILE}" \
     -n ${INSTALLER_NAMESPACE}
+oc ${hub} label secret osac-remote-kubeconfig \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)